pkgs/stdenv/generic/default.nix

```diff
@@ -1,4 +1,6 @@
-let lib = import ../../../lib; stdenv-overridable = lib.makeOverridable (
+lib:
```

Comment:
Why is this passed as a positional argument rather than as a regular argument? This seems very non-idiomatic.

Reply:
That file defines an overridable function, so it needs to have access to `lib` before it accepts other arguments